src/renderer/components/Settings/Advanced.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { SettingsStoreState } from '../../stores/settings'
+import Encryption from './Encryption'
+import { useTranslationFunction } from '../../contexts'
+import { ExperimentalFeatures } from './ExperimentalFeatures'
+import ImapFolderHandling from './ImapFolderHandling'
+import ManageKeys from './ManageKeys'
+import SettingsHeading from './SettingsHeading'
+import SettingsSeparator from './SettingsSeparator'
+
+type Props = {
+  settingsStore: SettingsStoreState
+}
+
+export default function Advanced({ settingsStore }: Props) {
```

Comment:
I renamed all settings-related components from `Settings-Advanced.tsx` to `Settings/Advanced.tsx`